src/gradualizer_prelude.erl

```diff
@@ -170,3 +170,50 @@
 -spec proplists:lookup_all(any(), list()) -> [tuple()].
 -spec proplists:append_values(any(), list()) -> list().
 -spec proplists:substitute_aliases([{any(), any()}], list()) -> list().
+
+%% qlc
+
+-type(cache() :: ets | list | no).
+-type(key_pos() :: pos_integer() | [pos_integer()]).
+-type(max_list_size() :: non_neg_integer()).
+-type(tmp_file_usage() :: allowed | not_allowed | info_msg
+                        | warning_msg  | error_msg).
+-type(tmp_directory() :: [] | file:name()).
+
+-spec qlc:e(query_handle_or_list()) ->
```

Comment:
I don't see type `query_handle_or_list()` defined

Reply:
Yeah, that was a typo. I've already fixed this